src/main.cpp

```diff
@@ -27,6 +27,8 @@
 #include "Cafe/IOSU/legacy/iosu_crypto.h"
 #include "Cafe/OS/libs/vpad/vpad.h"
 
+#include "Common/platform.h"
+
```

Comment:
Is this include necessary for compilation on GCC?

Reply:
> Is this include necessary for compilation on GCC?

I believe main.cpp uses __declspec or other windows specific functions which are fixed using macros in precompiled.h

Reply:
I think it requires unix/platform which is included in Common/platform. Will double check.

Reply:
Looks okay w/out. Removed here https://github.com/tomlally/Cemu/commit/3d28b5e0142ad641ac68887ceaff54adaf6c0ae3